.gitignore

```diff
@@ -7,8 +7,6 @@ pids
 *.pid
 *.seed
 
-/dist/
```

Comment:
I'd prefer to add the build script to `npm install` so that it will build automatically.